tools/ontograph

```diff
@@ -209,6 +216,13 @@ def main(
     )
     args = parser.parse_args(args=argv)
 
+    if args.leafs:
+        warnings.warn(
+            "The `--leafs` option is deprecated and will soon be removed."
+            "Please use `--leaves` instead."
```

Comment:
```suggestion
            "Please use `--leaves` instead.",
            DeprecationWarning,
```
Not sure this is correct and also whether `DeprecationWarning` is imported by default (like, e.g., `Exception`). But in this way the warning will be a "proper" deprecation warning and not a `UserWarning`, which is the default warning class.